server/src/main/kotlin/suwayomi/tachidesk/server/ServerSetup.kt

```diff
@@ -236,14 +236,25 @@ fun applicationSetup() {
             serverConfig.socksProxyEnabled,
             serverConfig.socksProxyHost,
             serverConfig.socksProxyPort,
-        ) { proxyEnabled, proxyHost, proxyPort ->
-            Triple(proxyEnabled, proxyHost, proxyPort)
+            serverConfig.socksProxyUsername,
+            serverConfig.socksProxyPassword,
+        ) { proxyEnabled, proxyHost, proxyPort, proxyUsername, proxyPassword ->
+            data class DataClassForDestruction(
+                val proxyEnabled: Boolean,
+                val proxyHost: String,
+                val proxyPort: String,
+                val proxyUsername: String,
+                val proxyPassword: String,
+            )
+            DataClassForDestruction(proxyEnabled, proxyHost, proxyPort, proxyUsername, proxyPassword)
```

Comment:
not sure if this is the best way to convert multiple values to one, kotlin doesn't have a Quintuple type.